code/_onclick/click.dm

```diff
@@ -93,7 +93,7 @@
 
 		return
 
-	if(!isturf(loc) && !is_holder_of(src, A)) // Can't touch anything from inside a locker/sleeper etc, unless it's inside our inventory.
+	if(A != loc && !isturf(loc) && !is_holder_of(src, A)) // Can't touch anything from inside a locker/sleeper etc, unless it's inside our inventory.
```

Comment:
We need to check this - it was the topic of a lot of discussion a while ago about what you could click.